contracts/rateproducer/include/rateproducer.hpp

```diff
@@ -44,7 +44,7 @@ using eosio::public_key;
 */
 namespace eosio {
     constexpr name system_account{"eosio"_n};
-    constexpr name eden_account{"genesis.eden"_n};
+    constexpr name eden_account{"genesisdeden"_n};
```

Comment:
use `genesis.eden` instead of `genesisdeden`

Reply:
I am a bit confused, aren't we supposed to use `genesisdeden `because we are using jungle 4 account?

Reply:
mainnet: `genesis.eden`
tesnet: `genesisdeden`

we are trying to get the `genesis.eden` account name on testnet but not yet